src/rock-paper-scissors/services/game.service.ts

```diff
@@ -0,0 +1,161 @@
+import { Injectable } from '@nestjs/common';
+import { GameModel } from './game.model';
+import { Game } from '../schemas/game.schema';
+import { GameItem, GameRound, PlayerChoice } from '../types/game';
+import { AddChoiceToRoundDto } from '../dto/add-choice-to-round.dto';
+import { GameProcess } from './game-process/game-process.service';
+
+@Injectable()
+export class GameService {
+  readonly NO_ONE_WIN = 'noOneWin';
```

Comment:
Let's be ready for current sensibilities, better say TIE :D

Reply:
Sure.